src/coreclr/src/vm/stubmgr.cpp

```diff
@@ -1844,6 +1844,12 @@ BOOL ILStubManager::TraceManager(Thread *thread,
         trace->InitForUnmanaged(target);
     }
 #endif // FEATURE_COMINTEROP
+    else if (pStubMD->IsStructMarshalStub())
+    {
+        // There's no "target" for struct marshalling stubs
+        // so we have nowhere to tell the debugger to move the breakpoint.
+        return FALSE;
```

Comment:
My only question here would be what changed that caused this failure?

Reply:
We just happened to find the corner case that would hit this failure. This has been a latent bug ever since the struct marshal stubs were introduced. When we used to use the native "field marshalers" before https://github.com/dotnet/coreclr/pull/26340 there weren't any struct marshal stubs.